app/src/editor/canvas/hooks/useCanvasNotifications.jsx

```diff
@@ -65,6 +65,26 @@ export function pushErrorNotification(error) {
     })
 }
 
+export function pushWarningNotification(warning) {
+    console.warn(warning) // eslint-disable-line no-console
+    // eslint-disable-next-line react/no-danger
```

Comment:
Disabling [`react/no-danger`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-danger.md) shouldn't be necessary here.

Reply:
Fixed in 97a001c